ecl2df/summary.py

```diff
@@ -694,6 +695,14 @@ def summary_reverse_main(args):
     summary_df = pd.read_csv(args.csvfile)
     logger.info("Parsed %s", args.csvfile)
 
-    eclsum = df2eclsum(summary_df, args.output)
+    outputdir = Path(args.output).parent
+    eclbase = Path(args.output).name
+
+    # EclSum.fwrite can only write to current directory:
+    cwd = os.getcwd()
+    os.chdir(outputdir)
+    eclsum = df2eclsum(summary_df, eclbase)
     EclSum.fwrite(eclsum)
+    os.chdir(cwd)
```

Comment:
I suggest to use a `contextmanager` to handle returning to the original directory - although no `contextmanager` will stand a chance against a `util_abort`.

I understand this is a limitation in ecl, but it should rather be fixed there (I had a short glance, and it doesn't seem to be trivial to fix as ecl sets the casename on the c-object when wanting to write, instead of passing a path)

Reply:
I thought of a context manager, but failed to see the upside in this situation?